magellan-server does not depend on tslib (but requires)

# Bug Report

When using magellan-server `serve` function, `tslib` is required but can't be found. It should be added to the `package.json` dependencies.


### 🔎 Search Terms

tslib

### 🕗 Version & Regression Information

* no blocker, I can run `yarn add tslib`

### 💻 Code

```sh
echo '{}' > package.json
yarn add "@quatico/magellan-server@0.1.4"
node -p 'require("@quatico/magellan-server").serve()'
```

### 🙁 Actual behavior

💣 

```text
❯ node -p 'require("@quatico/magellan-server").serve()'
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module 'tslib'
Require stack:
- /private/tmp/magrep/node_modules/@quatico/magellan-server/lib/configuration/configuration-repository.js
- /private/tmp/magrep/node_modules/@quatico/magellan-server/lib/configuration/index.js
- /private/tmp/magrep/node_modules/@quatico/magellan-server/lib/index.js
- /private/tmp/magrep/[eval]
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/private/tmp/magrep/node_modules/@quatico/magellan-server/lib/configuration/configuration-repository.js:10:17)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/private/tmp/magrep/node_modules/@quatico/magellan-server/lib/configuration/configuration-repository.js',
    '/private/tmp/magrep/node_modules/@quatico/magellan-server/lib/configuration/index.js',
    '/private/tmp/magrep/node_modules/@quatico/magellan-server/lib/index.js',
    '/private/tmp/magrep/[eval]'
  ]
}
```

### 🙂 Expected behavior

```text
❯ node -p 'require("@quatico/magellan-server").serve()'
/private/tmp/magrep/node_modules/@quatico/magellan-server/lib/server.js:26
    const { port = (0, exports.normalizePort)(process.env.PORT) } = options;
            ^

TypeError: Cannot read properties of undefined (reading 'port')
```

--- a/Release-Notes.md
+++ b/Release-Notes.md
@@ -7,6 +7,14 @@
 
 # Releases
 
+## v0.1.5
+
+> TBA
+
+### TBD
+
+- Improvement :gift_heart:: magellan-server has tslib as production dependency to ease integration in custom servers.
+
 ## v0.1.4
 
 > 2022-11-08

--- a/packages/server/package.json
+++ b/packages/server/package.json
@@ -44,7 +44,8 @@
         "morgan": "~1.9.1",
         "multer": "1.4.5-lts.1",
         "node-fetch": "^2.6.7",
-        "socket.io-client": "^2.5.0"
+        "socket.io-client": "^2.5.0",
+        "tslib": "^2.3.1"
     },
     "devDependencies": {
         "@types/cookie-parser": "^1.4.2",
@@ -74,7 +75,6 @@
         "supertest": "^6.1.6",
         "ts-jest": "^27.0.7",
         "ts-node": "^10.4.0",
-        "tslib": "^2.3.1",
         "typescript": "~4.7.4"
     },
     "engines": {

--- a/packages/test/package.json
+++ b/packages/test/package.json
@@ -47,7 +47,6 @@
         "@quatico/magellan-server": "^0.1.4",
         "@quatico/magellan-shared": "^0.1.4",
         "@types/node": "16",
-        "tslib": "^2.3.1",
         "typescript": "~4.7.4"
     },
     "engines": {

